bundles/jsch-agentproxy-jsch/pom.xml

```diff
@@ -82,7 +82,8 @@ limitations under the License.
             </goals>
             <configuration>
               <!-- the RAT plugin complains if we use the default location and we don't want it in the SCM -->
-              <dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>
+              <!--<dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>-->
```

Comment:
tested with that line reverted and it works as well. Can't remember what I was testing.